### PR TITLE
[Telemetry Table] Display applied filters

### PR DESCRIPTION
**Pull request by psarram**
Display a list of applied filters at the bottom of the telemetry table. If filters have mixed values, display 'Mixed' also.

### CODE REVIEW

**Comment by charlesh88** [comment]
###  Style related changes:
- CSS, markup;
- Added dynamic labeling and titling for mixed/non-mixed filter states;
- Theme colors defined and added;
- Added new filter icon glyphs for both 16px and 12px fonts;
- Revised/normalized font project and glyph file names;

**Comment by akhenry** [comment]
@charlesh88 @psarram This good to go?

**Comment by psarram** [comment]
@akhenry Yes, it's ready for review.

File: src/styles-new/sass-base.scss [hunk 1]
```diff
@@ -25,8 +25,8 @@
 // Do not include anything that renders to CSS!
 @import "constants";
 @import "constants-mobile.scss";
-//@import "constants-espresso"; // TEMP
-@import "constants-snow"; // TEMP
+@import "constants-espresso"; // TEMP
+//@import "constants-snow"; // TEMP
```
Comment (akhenry) [inline src/styles-new/sass-base.scss]: Can we keep the default snow for now? It's just one less thing to remember with VISTA deployments...

Comment (psarram) [reply src/styles-new/sass-base.scss]: @akhenry Thanks for the review! Changed the default back to snow.

**Comment by akhenry** [comment]
## Reviewer Checklist

* Changes appear to address issue? Y
* Appropriate unit tests included? N/A
* Code style and in-line documentation are appropriate? Y
* Commit messages meet standards? Y

**Pull request merged**